docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +79,19 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+
+.. _intro:troubleshooting:installation:quicksetup:
```

Comment:
Can we give this a more meaningful name, like `...:installation:postgresql-setup-autodetect-issues`?

Reply:
Sure, in general I thought that we should keep the "tag" names short, but I guess this is more important for anything that branches out, and end leaves can be longer.